### ISSUE TITLE
Instalment schedule SDK incompatibility with API

### ISSUE THREAD

**Issue by jakubskrzypek** [gocardless/gocardless-pro-java#70]
I am currently integrating with go-cardless via the Java library you've provided, however, I've come into an issue in regards to instalment schedule behaviour. Data sent to POST '/instalment_schedules' endpoint must be wrapped in an "instalment_schedules" envelope but the SDK wraps in 'data'.

```json
{
    "error": {
        "message": "Data sent to this endpoint must be wrapped in a 'instalment_schedules' envelope",
        "errors": [
            {
                "reason": "invalid_document_structure",
                "message": "Data sent to this endpoint must be wrapped in a 'instalment_schedules' envelope"
            }
        ],
        "documentation_url": "https://developer.gocardless.com/api-reference#invalid_document_structure",
        "type": "invalid_api_usage",
        "request_id":  "",
        "code": 400
    }
}
```

**Comment by vdandugc** [comment]
Hi, Thanks for reporting this, we will look into this.

**Comment by vdandugc** [comment]
This should be fixed by version [4.9.1](https://github.com/gocardless/gocardless-pro-java/releases/tag/v4.9.1).
